docs/guide/extending_modules.md

```diff
@@ -112,7 +112,6 @@ Optionally, Universal Discovery module can take a following list of props:
 - **labels** _{Object}_ - a hash containing text messages to be placed across many places in a component. It contains text labels for child components, see [universal.discovery.module.js](https://github.com/ezsystems/ezplatform-admin-ui-modules/blob/master/src/modules/universal-discovery/universal.discovery.module.js#L438) for details,
 - **selectedItemsLimit** _{Number}_ - the limit of items that can be selected. Should be combined with the `multiple` attribute set to `true`. Default value is `0`, which means no limit,
 - **allowContainersOnly** _{Boolean}_ - when true, only containers can be selected. Default value: `false`,
-- **onlyContentOnTheFly** _{Boolean}_ - when true, only Content on the Fly is shown in the UDW. Default value: `false`,
```

Comment:
What about `cotfForcedLanguage` below?

Reply:
https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/202/files#diff-d126f4474e04d36664723db868ec0d2eL1146

Reply:
Both moved to `ez_platform_v3.0_deprecations.md`